### DISCUSSION TITLE
Downgrade to previous version

### DISCUSSION THREAD

**Discussion by domix75** [goastian/midori-desktop#90] [Q&A] [answered]
since I installed Midori 11.4.2 (debian 64), it takes > 100% processor resource and gets unusable. I have many tabs open but that was the case with 11.3. Is there a way to get the previous 11.3 debian package to downgrade and wait for a new version ?

**Comment by abejaranoh** [comment, marked as answer]
older versions can be found here: 
https://github.com/goastian/midori-desktop/releases

**Comment by domix75** [comment]
Thanks a lot !
This solved my pbm. i don't have time to investigate further. 
I'll try out next version when it comes
